<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSnapOne">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPicOne">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>加载图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalibra">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>基准点</string>
   </property>
   <widget class="QPushButton" name="pushButton_StdW">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225,225,225);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>拍照位置</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_wStdx">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_wStdy">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ShangStdx">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ShangStdy">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ShangStd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225,225,225);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>作业位置</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_pStdy">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_pStdx">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stdpix">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225,225,225);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>图像位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OffsetValue">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225,225,225);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>偏移范围</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_offsetY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_offsetX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="CalibrationCoordinate">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>标定点坐标</string>
   </property>
   <widget class="QComboBox" name="PointSelector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(225,225,225);
border-radius:10px;</string>
    </property>
    <item>
     <property name="text">
      <string>请选择标定点</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标定点9</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="YCoordinate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="XCoordinate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_Data">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Test">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>1001</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提示：1.Mark点坐标（拍照位置）与运行时的螺丝拍照位置必须为同一台产品不移动时的数据，若移动产品更改Mark点拍照位置，则必须调整所有螺丝拍照位置；反之亦然。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>1001</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>      2.偏移范围为正负值，如设置偏移量为5，则运行时对比范围为±5mm(设置时需要设为正值)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
